docs/pages/architecture/proxy-peering.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Proxy Peering
+description: How Teleport implements more efficient networking with Proxy Peering.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  Proxy Peering is available starting from Teleport `10.0`.
+</Details>
+
+Proxy Peering enables an agent to be reachable without it connecting to every
+Teleport Proxy instance. This allows Teleport Proxy instances to scale horizontally
+without increasing the number of connections created by agents.
+
+Proxy Peering is a type of tunnel strategy configurable in Teleport. A tunnel
```

Comment:
I think we can remove this paragraph. The next paragraph explains how Proxy Peering works in relation to the Proxy Service's default behavior, and the previous paragraph explains the benefit.